state-chain/traits/src/liquidity.rs

```diff
@@ -12,20 +28,31 @@ pub trait LpDepositHandler {
 	fn add_deposit(who: &Self::AccountId, asset: Asset, amount: AssetAmount) -> DispatchResult;
 }
 
-pub trait LpBalanceApi {
+/// API for interacting with the liquidity provider pallet.
+pub trait LpApi {
```

Comment:
LpApi is a little too generic and clashes with the actual lp api. 

How about `LpRegistration`?